{{cookiecutter.project_slug}}/clients/web/vue3/tests/e2e/specs/test-login.cy.ts

```diff
@@ -6,6 +6,7 @@ describe('Tests login workflow', () => {
   })
 
   it('Filling in email and passwords goes to', () => {
+    cy.visit('/login')
```

Comment:
Has this not happened in the previous it statement?

Reply:
I think cypress changed the way they run the test. They clean up in between. It starts back at home

Reply:
(This upgraded cypress from 8 -> 13 )

Reply:
Ah great, that makes sense